js/viewer.js

```diff
@@ -163,8 +163,13 @@ function redirectImages(node) {
 
 function applyAnchor() {
     var anchors = document.getElementsByName(setup.anchor);
-    if (anchors.length > 0)
+    if (anchors.length > 0) {
         anchors[0].scrollIntoView(true);
+        if (!local)
+          window.scrollBy(0, -46); // 46 is the height of the header of Cyberbotics web page
+        else
+          window.scrollBy(0, 180); // manual adjustment for the off-line version
```

Comment:
No, I don't have any clue about this.
The element.scrollIntoView() function seems to be called correctly, and should work without any offset.
https://developer.mozilla.org/fr/docs/Web/API/Element/scrollIntoView

Hum, the note in this page indicating:

> L'élément peut ne pas être aligné complètement avec le haut ou le bas, selon la disposition des autres éléments.

On this link, they are recommending to use window.scrollTo() instead:
http://stackoverflow.com/questions/8922107/javascript-scrollintoview-middle-alignment